package.json

```diff
@@ -19,6 +19,7 @@
     "compile": "babel src --out-dir dist -s --source-maps --extensions '.ts,.tsx' --ignore src/**/__tests__",
     "kitchen-sink": "cd examples/kitchen-sink && yarn start",
     "lint": "tslint -c tslint.json --project tsconfig.json",
+    "postinstall": "yarn compile",
```

Comment:
not sure if it is just me but got some error on installing the latest package

```
[8/14] ⠐ @apollo/rover
warning Error running install script for optional dependency: "/Users/roderickhsiao/git/kindred-landing/node_modules/cpu-features: Command failed.
Exit code: 127
Command: node buildcheck.js > buildcheck.gypi && node-gyp rebuild
Arguments: 
[7/14] ⠄ sharp
[11/14] ⠄ cypress
[13/14] ⠄ ssh2
[8/14] ⠄ @apollo/rover
error /Users/roderickhsiao/git/kindred-landing/node_modules/@artsy/fresnel: Command failed.
Exit code: 127
Command: yarn compile
Arguments: 
Directory: /Users/roderickhsiao/git/kindred-landing/node_modules/@artsy/fresnel
Output:
yarn run v1.16.0
warning ../../../../package.json: No license field
$ babel src --out-dir dist -s --source-maps --extensions '.ts,.tsx' --ignore src/**/__tests__
```

Reply:
Ah, lemme fix that

Reply:
Fixed in https://github.com/artsy/fresnel/pull/345

Reply:
Lightening speed thank you!